apps/experiments/migrations/0034_update_AWS_synthetic_voices.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 4.2 on 2023-10-11 15:18
+import os, json
+from django.db import migrations
+
+
+def load_language_code(apps, schema_editor):
```

Comment:
Since these have already been loaded I think you can just run `SyntheticVoice.objects.update(service="AWS")`

Reply:
As per my understanding the issue is that the language_code isn't loaded on the existing aws voices, so we need to 1. fetch the correct synthetic voice entry using the other fields (name, language, gender etc) and 2. update it's specific language code. This relates to the reason why the two migrations cannot be merged i.e. the language code should first be populated before locking it down to null=False.

If we really want only one migration, I'm wondering if we shouldn't add a default nonsense language code, and update it in a subsequent data migration? It feels like a bad approach though. Not sure if there are other better approaches to handling this @snopoke ?

Reply:
Ah right, language code... To keep it in one migration you can add it as a nullable field and then add in the 'not null' constraint.

Reply:
As opposed to doing it with the default that doesn't get preserved when I add the field? The issue there is that I can't make it a required field without the default.